heimdall/source/BeginSessionPacket.h

```diff
@@ -28,11 +28,26 @@ namespace Heimdall
 {
 	class BeginSessionPacket : public SessionSetupPacket
 	{
+		protected:
+
+			enum
+			{
+				kDataSize = SessionSetupPacket::kDataSize + 4
+			};
+
 		public:
 
 			BeginSessionPacket() : SessionSetupPacket(SessionSetupPacket::kBeginSession)
 			{
 			}
+
+			void Pack(void)
+			{
+				SessionSetupPacket::Pack();
+
+				/* Odin protocol verison. */
```

Comment:
just a cosmetic thingy, but there is spelling mistake "verison" in the comment

Reply:
Right, I will update. Do you otherwise think this is ok or is there anything else to change?